lonestar/analytics/cpu/betweennesscentrality/BetweennessCentralityOuter.cpp

```diff
@@ -36,10 +36,11 @@ static const char* url = "betweenness_centrality";
 static llvm::cl::opt<std::string> inputFile(llvm::cl::Positional,
                                             llvm::cl::desc("<input file>"),
                                             llvm::cl::Required);
-static llvm::cl::opt<int> iterLimit("limit",
-                                    llvm::cl::desc("Limit number of iterations "
-                                                   "to value (0 is all nodes)"),
-                                    llvm::cl::init(0));
+static llvm::cl::opt<unsigned int>
+    iterLimit("numOfOutSources",
+              llvm::cl::desc("Number of sources WITH EDGES "
```

Comment:
I would numOfSources to be consistent with other apps. I would prefer with/without edges to be consistent across apps.

Reply:
Not all BC apps support numOfOutSources at the moment. I can see about adding that support in to the ones that don't have it.

Outer doesn't have numOfSources as it doesn't make sense to give a thread empty work (outer assigns individual sources to threads).

Reply:
Can we also add a command-line argument to pass multiple sources instead of just reading from the file? For example, -numOfOutSources=4 -sources=2,3,44,5.

Reply:
opened at #286

Reply:
> I would prefer numOfSources to be consistent with other apps. I would prefer with/without edges to be consistent across apps.

That's true for other apps as well. Just be consistent among all of them. For now, if I'm fine if it considers vertices without edges.